plugins/topology/manifests/clusterrole.yaml

```
@@ -0,0 +1,83 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: janus-idp-topology-plugin
+rules:
+  - apiGroups:
+      - ''
+    resources:
+      - pods
+      - services
+      - configmaps
+      - limitranges
+    verbs:
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - metrics.k8s.io
+    resources:
+      - pods
+    verbs:
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - apps
+    resources:
+      - daemonsets
+      - deployments
+      - replicasets
+      - statefulsets
+    verbs:
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - autoscaling
+    resources:
+      - horizontalpodautoscalers
+    verbs:
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - networking.k8s.io
+    resources:
+      - ingresses
+    verbs:
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - batch
+    resources:
+      - jobs
+      - cronjobs
+    verbs:
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - tekton.dev
+    resources:
+      - pipelineruns
+      - taskruns
+    verbs:
+      - get
+      - list
```

```suggestion
  - apiGroups:
      - tekton.dev
    resources:
      - pipelines
      - pipelineruns
      - taskruns
    verbs:
      - get
      - list
```

we can add `pipelines` here and can remove https://github.com/janus-idp/backstage-plugins/pull/502/files#diff-d9f3781aaa2ba65a79b4a25d4ae701a63dc4dd648eb19b6eee9ebe063eb7647cR76-R83